### DISCUSSION TITLE
Incluir en el README pasos mínimos para instalación y ejecución del proyecto

### DISCUSSION THREAD

**Discussion by ReynerHL** [phpcfdi/app-descarga-masiva#8] [Ideas]
Considero que sería buena idea poner en el README los pasos básicos para facilitar la instalación y la ejecución del proyecto. Esto abre la posibilidad de cooperación a desarrolladores menos experimentados con las tecnologías propuestas y que están en disposición de aportar en el mismo tiempo que se superan con dichas tecnologías.

Si esta idea procede, puedo elaborar un pull request con una sección en el README para tal fin.

**Comment by luffynando** [comment]
Siguiendo las lineaciones, propuestas por @eclipxe13, la guía de contribución se encuentra en el archivo [CONTRIBUTING.md](https://github.com/phpcfdi/app-descarga-masiva/blob/main/CONTRIBUTING.md)
